Source/Smocks/IL/AssemblyRewriter.cs

```diff
@@ -122,6 +122,9 @@ public IAssemblyLoader Rewrite(string path)
                 WriteSymbols = hasSymbols
             };
 
+            foreach (var module in assembly.Modules)
+              module.Mvid = Guid.NewGuid();
```

Comment:
For consistency, please fix the indenting to match the current code base convention of 4 spaces. Also, I prefer to use curly braces (each on a new line) for _every_ for each body, including the single line ones.